### ISSUE TITLE
Refrigerator reading wrong fridge/freezer temp

### ISSUE THREAD

**Issue by electrosparkz** [ollo69/ha-smartthinq-sensors#64]
When I initially installed the integration, my fridge and freezer temp sensors were reading 10F and 8F. This never changed the whole time I had the integration set up. I realized later that it was stated the integration wouldn't work with an account using google or otherwise, so I followed the steps to disassociate the fridge, create a new LG only account, readd the fridge, then set the integration back up authing with the LG account. It's getting data, because the door open sensor works just fine, but the temps are still reading 10 and 8. My setpoints are 37 and 0. I looked through the refrigerator module source, and couldn't see any real reason where the 8 and 10 would be coming from, unless the data was bad. I'm not sure how to debug these modules much further to know what the raw return data from the LG API is, so any help would be appreciated.

**Comment by ollo69** [comment]
Can you post here the JSON info for your refrigerator? You should fing the link to download it in the HA log during start-up.
Probably I'm not reading properly the information for your refrigerator, temperature mapping is quite complex...

**Comment by electrosparkz** [comment]
I can't seem to find the info for the fridge. The only reference to the smartthinq module, is the warning about it being a custom integration:

2020-07-04 11:34:59 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for smartthinq_sensors which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.

There's no other reference to smartthinq or lg that I can find in any of the logs.

**Comment by ollo69** [comment]
You must open the log file `home-assistant.log` inside homeassistant home directory, There you will find a info line log like this:

```
Loading model info for Refrigerator. Model: ..., Url:...
```

I need the Url indicated there.

**Comment by electrosparkz** [comment]
There really just isn't anything there aside from that initial log line for being a custom component:

`hass@utilibox:~/hassos_smb$ cat home-assistant.log | grep thinq
2020-07-04 11:34:59 WARNING (MainThread) [homeassistant.loader] You are using a custom integration for smartthinq_sensors which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant.
hass@utilibox:~/hassos_smb$ cat home-assistant.log | grep model
hass@utilibox:~/hassos_smb$ cat home-assistant.log | grep -i refrigerator
`

I know the module is working, since the fridge open/close is reporting accurately. I'm not sure why it's not logging anything pertaining to loading the model info. I even tried adding the module to the logger config set to debug.

Do you have any suggestions as to why I'm not seeing anything in the log, or where else to look?

![fridge_open_close](https://user-images.githubusercontent.com/10894111/86520827-93aa2b80-bdfd-11ea-9f1e-2349554f0e16.png)

**Comment by ollo69** [comment]
If you have nothing probably is because your log level is set to warning and you don't see info.
Try adding this line to configuration.yaml and reboot:

```
logger:
  default: info
```

**Comment by electrosparkz** [comment]
That did it. Here are all the log entries I found:

2020-07-04 17:12:57 INFO (MainThread) [custom_components.smartthinq_sensors] SmartThinQ client connected
2020-07-04 17:12:57 INFO (MainThread) [custom_components.smartthinq_sensors] Starting LGE ThinQ devices...
2020-07-04 17:12:57 INFO (SyncWorker_9) [custom_components.smartthinq_sensors.wideq.core_v2] Loading model info for Refrigerator. Model: 2REF11EID___2, Url: https://aic.lgthinq.com:46030/api/webContents/modelJSON?modelName=2REF11EID___2&countryCode=WW&contentsId=JS0903071724262497&authKey=thinq

2020-07-04 17:12:58 INFO (MainThread) [custom_components.smartthinq_sensors] LGE Device added. Name: Refrigerator - Type: REFRIGERATOR - Model: 2REF11EID___2 - Mac: 20:3d:bd:45:30:fe - ID: 4519afb0-518a-11e8-bee3-203dbd4530fe
2020-07-04 17:12:58 INFO (MainThread) [custom_components.smartthinq_sensors] Founds 1 LGE device(s)
2020-07-04 17:12:58 INFO (MainThread) [homeassistant.components.sensor] Setting up sensor.smartthinq_sensors
2020-07-04 17:12:58 INFO (MainThread) [homeassistant.components.binary_sensor] Setting up binary_sensor.smartthinq_sensors
2020-07-04 17:12:58 INFO (MainThread) [custom_components.smartthinq_sensors.sensor] Starting LGE ThinQ sensors...

**Comment by ollo69** [comment]
Fine, I will take a look. I removed sensitive info from your msg

**Comment by KTibow** [comment]
@ollo69 remember to delete the previous history

**Comment by electrosparkz** [comment]
Thank you for removing that, and thank you for taking a look!

**Comment by ollo69** [comment]
Please can you post here the attribute of your refrigerator like this;

![image](https://user-images.githubusercontent.com/60491700/86530068-cf083100-beb5-11ea-83e4-77b49fd0cea6.png)

**Comment by electrosparkz** [comment]
<img width="312" alt="Screen Shot 2020-07-05 at 8 55 23 AM" src="https://user-images.githubusercontent.com/10894111/86536547-67de8280-be9d-11ea-8b7e-2c5ab7a502d1.png">

**Comment by ollo69** [comment]
This issue should be fixed by last release ([v0.5.4](https://github.com/ollo69/ha-smartthinq-sensors/releases/tag/v0.5.4))
Please check and let me know

**Comment by electrosparkz** [comment]
That did it. Thanks!
